components/content/List/List.css

```diff
@@ -0,0 +1,70 @@
+diamond-list {
```

Comment:
This needs to support dl as well

valid markup for dl includes wrapping the dt/dd in a div, we could force that when used inside this component if it's easier

Reply:
DLs feel like quite a different use case? The markup is totally different. What would we do with the dl/dt?

Reply:
Would they benefit from their own component?

Reply:
Its helpful for outputting details or data with key value pairs

For example in RL it's used to output an LAs address, website, phone number

It is a list element so I think this component still makes sense even if it doesn't modify anything but the spacing on the separating divs and removed the padding/margin and bold dt text for unstyled ones